src/styles/block/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $main-borderCard-color;
+  border-radius: 5px;
+  background-color: $main-backgroundCard-color;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $main-textCard-color;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    color: $secondary-textCard-color;
+  }
+
+  &__feedback-container {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__feedback-reviews {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    height: 16px;
```

Comment:
<img width="614" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/e3e6fe76-af3e-4c18-b736-d620af40c83c">
you can just use padding instead of the height